wagtail/admin/action_menu.py

```diff
@@ -44,7 +43,6 @@ def is_shown(self, context):
             'view' = 'create', 'edit' or 'revisions_revert'
             'page' (if view = 'edit' or 'revisions_revert') = the page being edited
             'parent_page' (if view = 'create') = the parent page of the page being created
-            'user_page_permissions' = a UserPagePermissionsProxy for the current user, to test permissions against
```

Comment:
This needs to be removed as well:
https://github.com/wagtail/wagtail/blob/cba75351a8b9512f5fa2b89408879f59f7d064cd/docs/reference/hooks.md?plain=1#L817